### DISCUSSION TITLE
Hud tint not being transfered after changing turrets

### DISCUSSION THREAD

**Discussion by AbyssBeast27** [FakeFishGames/Barotrauma#13655] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Hud tint not being transfered after changing turrets/hardpoints unlike its colouring and light colour 

### Reproduction steps

1. Make a turret/hardpoint with edited hud tint
2. Start a campaign
3. Change hardpoint/turret with edited hud tint 

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

Other

### -

Happens on both unstable/stable versions

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Thank you for your report.

Was able to reproduce this with custom barsuk where i modified one turrets hud tint to be bright red and other turrets spot light hue to blue and blinking.
When i change out the turrets in first outpost, only the hue change is visible in next level

Custom sub for testing:
[BarsukTINTED.zip](https://github.com/user-attachments/files/22538998/BarsukTINTED.zip)

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/5afa6bdb97be324fb9967d5222a0b3c734177fe4, fix coming up in the next unstable build and the upcoming release.
